Classes/Controller/Cart/OrderController.php

```diff
@@ -60,10 +60,7 @@ public function initializeCreateAction()
             if (!$this->arguments->hasArgument($argumentName)) {
                 continue;
             }
-            if ($this->settings['validation'] &&
-                $this->settings['validation'][$argumentName] &&
-                $this->settings['validation'][$argumentName]['fields']
-            ) {
+            if ($this->settings['validation'][$argumentName]['fields'] ?? null) {
```

Comment:
Can you change this to isset in if-conditions? I think isset is more readable than the Null Coalescing Operator with null as fallback, isn't it?